apps/wiki-sonia/wiki-sonia.argoapp.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: sonia
+  namespace: argocd
+  annotations:
+    argocd.argoproj.io/sync-wave: "2"
+spec:
+  project: default
+  destination:
+    server: https://cedille.kubernetes.omni.siderolabs.io?cluster=k8s-cedille-production
+    namespace: sonia
+  source:
+    repoURL: https://github.com/ClubCedille/k8s-cedille-prod
+    path: apps/sonia/prod
```

Comment:
folder is named wiki-sonia but lets change it to 
sonia/wiki/prod 
like mentioned earlier